### PR TITLE
Comparison tool static content when no comparison

### PR DESCRIPTION
**Pull request by aniketsat**

### CODE REVIEW

File: components/comparision/FaqAccordion.vue
Comment (dhaladitya108) [inline components/comparision/FaqAccordion.vue:1]: We may directly use the existing component used in page: https://formester.com/solutions/quiz/

File: pages/comparison-tool.vue
Comment (dhaladitya108) [inline pages/comparison-tool.vue:1]: Creating a separate component to render the static content would be better.

File: components/comparision/FaqAccordion.vue
Comment (aniketsat) [reply components/comparision/FaqAccordion.vue:1]: Styles are different in this page

File: pages/comparison-tool.vue
Comment (aniketsat) [reply pages/comparison-tool.vue:1]: For each section separate components?

File: components/comparision/FaqAccordion.vue
Comment (dhaladitya108) [reply components/comparision/FaqAccordion.vue:1]: @aniketsat please verify with Ishan that we already have a different style for the FAQ. Should we replace it in all instances?

File: pages/comparison-tool.vue
Comment (dhaladitya108) [reply pages/comparison-tool.vue:1]: @aniketsat let's create a page to display all the static content. We could also create a component for sections that look similar.

**Pull request merged**